resources/views/livewire/project/shared/destination.blade.php

```diff
@@ -78,11 +78,11 @@ class="absolute bg-error -top-1 -left-1 badge "></div>
     </div>
     @if ($resource->getMorphClass() === 'App\Models\Application' && data_get($resource, 'build_pack') !== 'dockercompose')
         <div class="flex flex-col gap-2">
-            @if ($resource->persistentStorages()->count() > 0)
+            @if ($resource->fileStorages()->get()->filter(fn($storage) => $storage->content)->count() > 0)
```

Comment:
according to https://github.com/coollabsio/coolify/issues/7384#issuecomment-3592442169
showing a warning whenever any persistent storage is used may be an option instead of blocking the functionality:

```suggestion
            @if ($resource->persistentStorages()->count() > 0 || $resource->fileStorages()->count() > 0)
```